src/components/AddressInfo/index.tsx

```diff
@@ -42,7 +42,7 @@ interface Props {
   ethBalance?: string
 }
 
-const AddressInfo = ({ ethBalance, safeAddress, safeName }: Props) => {
+const AddressInfo = ({ ethBalance, safeAddress, safeName }: Props): JSX.Element => {
```

Comment:
Sorry for this but I replaced all those `JSX.Element` to `React.ReactElement`, could you update this? :trollface: